packages/react-components/src/components/ChatMessage/ChatMessageContent.tsx

```diff
@@ -198,6 +227,22 @@ const messageContentAriaText = (props: ChatMessageContentProps): string | undefi
     : undefined;
 };
 
+/* @conditional-compile-remove(image-gallery) */
+const defaultOnRenderInlineImage = (inlineImage: InlineImage): JSX.Element => {
+  return (
+    <img
+      {...inlineImage.imgAttrs}
```

Comment:
Do we want these to be last in the props so they override the defaults?

Reply:
Yes